front/app/api/areas/keys.ts

```diff
@@ -7,9 +7,9 @@ const baseKey = { type: 'area' };
 const areasKeys = {
   all: () => [baseKey],
   lists: () => [{ ...baseKey, operation: 'list' }],
-  list: (params: IAreasQueryParams) => [
-    { ...baseKey, operation: 'list', parameters: params },
-  ],
+  list: (
+    params: IAreasQueryParams | { endpoint: 'with_visible_projects_counts' }
+  ) => [{ ...baseKey, operation: 'list', parameters: params }],
```

Comment:
Not sure if `endpoint` should rather be part of an extension of area query params (a new type). Question is pending in your Slack DMs.